pyproject.toml

```diff
@@ -6,5 +6,6 @@ requires = [
 build-backend = "setuptools.build_meta"
 
 [tool.pyright]
+include = ["src"]
```

Comment:
This feels like an unrelated change.

Reply:
This is required for pyright to work. Otherwise "import xdsl" report as package xdsl not found.